apache2/ag_mdb/ag_mdb_internal.h

```diff
@@ -157,27 +157,27 @@ typedef unsigned long long PTR_OFFSET;
  ** @param shm_size: The size of the shared memory.
  ** @param entry_num:The number of entries in the shared memory.
  ** return: AGMDB_SUCCESS if success,
- **         AGMDB_ERROR if failed. 
+ **         AGMDB_ERROR if failed.
  */
-int SHM_init(PTR_VOID shm_base, unsigned int shm_size, int entry_num);
+int SHM_init(PTR_VOID shm_base, unsigned int shm_size, unsigned int entry_num);
```

Comment:
why this one will be in the format change?

Reply:
This is my fault. Because the function() in ag_mdb.cpp doesn't check ag_mdb_internal.h at all, it also works even when the definition in ag_mdb_internal.h is wrong. So I just revise it for the consistency. Do you think we need to separate it?